tajo-plan/src/main/java/org/apache/tajo/plan/util/PlannerUtil.java

```diff
@@ -133,11 +134,90 @@ public static boolean checkIfSimpleQuery(LogicalPlan plan) {
           }
         }
       }
+
+      if (!noWhere && scanNode.getTableDesc().isExternal() && scanNode.getTableDesc().getPartitionMethod() != null) {
```

Comment:
Yes, I developed handle both, but I found that Tajo does not support partitioned insert well.
So, I created the following issue, and change to handle external table only.
https://issues.apache.org/jira/browse/TAJO-1416